DiscordBot/bot.go

```diff
@@ -80,7 +80,7 @@ func handleSimplify(s *discordgo.Session, i *discordgo.InteractionCreate, data *
 	h.Set("Content-Disposition",
 		fmt.Sprintf(`form-data; name="%s"; filename="%s"`,
 			"image", quoteEscaper.Replace(attachment.Filename)))
-	h.Set("Content-Type", "image/png")
+	h.Set("Content-Type", res.Header.Get("Content-Type"))
```

Comment:
instead of setting some hardcoded value for what content type is we get a response and set the content type based on that . Looks good.